forms-flow-web/src/components/Modeler/ProcessTable.js

```diff
@@ -111,18 +143,21 @@ const ProcessTable = React.memo(() => {
   }, [search, dispatch, isBPMN]);
 
   const handleSort = (key) => {
-    setCurrentState((prevState) => ({
-      ...prevState,
-      sortConfig: {
-        ...prevState.sortConfig,
-        activeKey: key,
-        [key]: {
-          sortOrder: prevState.sortConfig[key]?.sortOrder === "asc" ? "desc" : "asc",
-        },
+    const newSortConfig = {
+      ...sortConfig,
+      activeKey: key,
+      [key]: {
+        sortOrder: sortConfig[key]?.sortOrder === "asc" ? "desc" : "asc",
       },
-    }));
+    };
+  
+    if (isBPMN) {
+      dispatch(setBpmSort(newSortConfig));
```

Comment:
here you are saving the sort into separate redux state right, so please review above comment